platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -174,6 +178,10 @@ MGL_EXPORT
  */
 @property (class, nonatomic, readonly) MGLOfflineStorage *sharedOfflineStorage;
 
+#pragma mark - Merging Offline Regions
+
+- (void)mergeOfflineRegions:(NSString *)databaseURL withCompletionHandler:(nullable MGLOfflineMergeCompletionHandler)completion;
```

Comment:
Decide whether the method will take an NSURL (as implied by `databaseURL`) or a string.

A more Cocoa name for this method would be either `-addContentsOfFile:withCompletionHandler:` or `-addContentsOfURL:withCompletionHandler:`. The primary effect of this method is to add offline packs; resolving conflicts (merging) is merely an edge case that can be documented in a documentation comment.